spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+
+  it 'exists' do
+    factory = FacilityFactory.new
+
+    expect(FacilityFactory.new).to be_an_instance_of(FacilityFactory)
+  end
+
+  it 'also exists' do
+    factory = FacilityFactory.new
+    co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+
+    expect(co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations).to be_an_instance_of(Array)
+  end
+
+  it 'takes the facility and arrays it into a Facility' do
+    factory = FacilityFactory.new
+    co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+    
+    expect(factory.create_facilities(co_dmv_office_locations)).to be_an_instance_of(Array)
+  end
+
+  it 'creates a NY facility' do
+    factory = FacilityFactory.new
+    new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+    ny_dmv_office_locations = DmvDataService.new.ny_dmv_office_locations
+
+    expect(factory.create_facilities(ny_dmv_office_locations)).to be_an_instance_of(Array)
+  end
+  
+  it 'creates a MO facility' do
+    factory = FacilityFactory.new
+    missouri_facilities = DmvDataService.new.mo_dmv_office_locations
+    mo_dmv_office_locations = DmvDataService.new.mo_dmv_office_locations
+
+    expect(factory.create_facilities(mo_dmv_office_locations)).to be_an_instance_of(Array)
+  end
+
+end
```

Comment:
We also want to get into that array that the method should be returning, and check that it is truly creating Vehicle objects as you'd expect it to. For starters, we could look at the first element of the array that was returned and make sure it was a vehicle object, and that each of it's attributes were not nil.